CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+- Add: -disableFileLog CLI parameter (ORION_DISABLE_FILE_LOG env var) to prevent Orion from logggin into a file

```

Comment:
Small typo:

```suggestion
- Add: -disableFileLog CLI parameter (ORION_DISABLE_FILE_LOG env var) to prevent Orion from logging into a file
```

(My fault... the typo was already in my original comment)